src/applications/simple-forms/20-10206/containers/IntroductionPage.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+
+import { focusElement } from 'platform/utilities/ui';
+import OMBInfo from '@department-of-veterans-affairs/component-library/OMBInfo';
+import FormTitle from 'platform/forms-system/src/js/components/FormTitle';
+import SaveInProgressIntro from 'platform/forms/save-in-progress/SaveInProgressIntro';
+
+class IntroductionPage extends React.Component {
```

Comment:
do you plan on using ConfirmationPageView and IntroductionPageView shared components for these? I know this is just what you get with yeoman generator

Reply:
Yes, @rhasselle-oddball , in the next Step \[Build frontend\].